datareduction/datareducer.cpp

```diff
@@ -22,15 +22,52 @@
 
 #include <cstdlib>
 #include <iostream>
+#include <string_view>
+#include "variable_dictionary.h"
+#include <optional>
 
 #include "datareducer.h"
 #include "../common.h"
+#include "datareductionoperator.h"
 #include "dro_populations.h"
 #include "../sysboundary/ionosphere.h"
 #include "../fieldtracing/fieldtracing.h"
 
 using namespace std;
 
+namespace {
```

Comment:
The spell checker (while nice!) should be its own PR.

Reply:
Oh right! I forgot this had leaked in here